txscript/script.go

```diff
@@ -624,6 +630,12 @@ const (
 	// executed.
 	ScriptDiscourageUpgradableNops
 
+	// ScriptVerifyCleanStack defines that the stack must contain only
+	// one stack element after evaluation and that the element must be
+	// true if interpreted as a boolean.
+	// This flag should never be used without the ScriptBip16 flag.
```

Comment:
Since this flag should never be used without the ScriptBip16 flag, the `NewScript` function should return an error when it's called with this flag set without also having the ScriptBip16 flag set.